prime-router/metadata/hl7_mapping/common/observation-result.yml

```
@@ -4,7 +4,7 @@ elements:
     hl7Spec: [ '%{hl7OBXField}-1' ]
 
   - name: result-value-type-st
-    condition: '(%resource.value.exists() and %resource.value.is(string)) or (%resource.dataAbsentReason.coding.where(extension(%`rsext-code-index-name`).value = "identifier").exists() and %resource.dataAbsentReason.coding.where(extension(%`rsext-code-index-name`).value = "identifier").code = "unknown")'
+    condition: '%resource.value is String'
```

After merging, due to the HAPI FHIR updates [here](https://github.com/CDCgov/prime-reportstream/pull/11406), this condition is no longer valid and smoke tests and integration tests are now failing. Kalish posted some tips [here](https://nava.slack.com/archives/C01HFHPNJEL/p1695841059693489) (not sure if the link is Nava-specific or if it'll work for you), but you should be able to just keep what's currently in `master` with the last part removed like before.

```suggestion
    condition: '%resource.value.exists() and %resource.value.is(string)'
```

---

Thankyou for the suggested fix!